apps/frontend/src/app/5_pages/ZeroPage/ZeroPage.tsx

```diff
@@ -235,8 +240,8 @@ export const ZeroPage: FC = () => {
                   <DialogBody>
                     <CloseCreditLine
                       onSubmit={handleTroveClose}
-                      creditValue={String(debt - LIQUIDATION_RESERVE_AMOUNT)}
-                      collateralValue={String(collateral)}
+                      creditValue={debt.sub(20)}
```

Comment:
```suggestion
                      creditValue={debt.sub(LIQUIDATION_RESERVE_AMOUNT)}
```

Reply:
No magic numbers please - either add it back or pull from `lib-base` instead, it's exposed as `ZUSD_LIQUIDATION_RESERVE`